remote-trait-object/src/forwarder.rs

```diff
@@ -63,10 +63,10 @@ impl ServiceForwarder {
         } else {
             let handlers = self.service_objects.read();
             crate::service::serde_support::port_thread_local::set_port(self.port.read().clone());
-            let result = handlers
-                .get(&object_id)
-                .unwrap_or_else(|| panic!("Fail to find {} from ServiceForwarder", object_id))
-                .dispatch_and_call(method, data);
+            let handler = Arc::clone(
+                handlers.get(&object_id).unwrap_or_else(|| panic!("Fail to find {} from ServiceForwarder", object_id)),
+            );
+            let result = handler.dispatch_and_call(method, data);
```

Comment:
Which deadlock is fixed? Could you tell me the deadlock?

Reply:
I had mistake. Please check the updated one